Samples.Objc/Samples.ObjC/SceneDelegate.m

```diff
@@ -7,40 +16,45 @@ @interface SceneDelegate ()
 @implementation SceneDelegate
 
 
-- (void)scene:(UIScene *)scene willConnectToSession:(UISceneSession *)session options:(UISceneConnectionOptions *)connectionOptions API_AVAILABLE(ios(13)) {
+- (void)scene:(UIScene *)scene willConnectToSession:(UISceneSession *)session options:(UISceneConnectionOptions *)connectionOptions {
   // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
   // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
   // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
+  
+  self.window = [[UIWindow alloc] initWithWindowScene:(UIWindowScene *)scene];
+  CheckoutViewController *viewController = [[CheckoutViewController alloc] init];
+  self.window.rootViewController = viewController;
+  [self.window makeKeyAndVisible];
 }
 
 
-- (void)sceneDidDisconnect:(UIScene *)scene API_AVAILABLE(ios(13)) {
```

Comment:
Why remove `API_AVAILABLE(ios(13))`?

Reply:
our deployment target is `iOS 11.0` which will show these errors:

![image](https://user-images.githubusercontent.com/7844550/112887003-874eb880-9087-11eb-8dc5-6a86194d5bea.png)